src/content/configuration/dev-server.md

```diff
@@ -844,6 +844,21 @@ module.exports = {
 };
 ```
 
+Sometimes you may want to use some options when opening the browser like opening an incognito window. In order to do that you can set `open` to an object. The object must have `app` property as an array. The first element in the array must be the browser name and the other following elements are the options you want. For example:
```

Comment:
This is very abstract in the beginning. Need to make it more of a documentation level e.g. remove `Sometimes...` Say clearly what you can achieve or what is allowed when passing object to `devServer.open`.

Is `app` the only key allowed in this object?

Reply:
what about : `If you want to use some switches when opening the browser like opening an incognito window (--incognito switch), you can set `open` to an object. The object must have `app` property as an array. The first element in the array must be the browser name and the other following elements are the switches you want. For example:`

No, all options in https://www.npmjs.com/package/open will work.

Reply:
Yes this sounds much better! Looks good, can you commit this?

Reply:
@EugeneHlushko Check my last commit 😄